asreview/simulation/simulate.py

```diff
@@ -212,9 +212,9 @@ def review(self):
             pbar_rel.close()
             pbar_total.close()
 
-            padded_results = list(self._results["label"]) + [0] * (
-                len(self.labels) - len(self._results["label"])
-            )
+            padded_results = list(
+                self._results.dropna(axis=0, subset="training_set")["label"]
+            ) + [0] * (len(self.labels) - len(self._results["label"]))
```

Comment:
You can resolve this with an if statement or with try-except (which is more Pythonic). Or a combination of both if you want to provide more informative feedback. 

```python
if len(set(padded_results)) < 2: 
    print("Can't compute loss and gain for labels with only relevant or irrelevant records")
```